scripts/move_tags_to_custom_fields.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import csv
+from closeio_api import Client as CloseIO_API
+
+parser = argparse.ArgumentParser(description='uploads custom fields from csv')
+parser.add_argument('csvfile', type=argparse.FileType('r'), help='csv file')
+parser.add_argument('--api_key', '-k', required=True, help='API Key')
+parser.add_argument('--development', '-d', action='store_true', help='Use a development (testing) server rather than production.')
+
+args = parser.parse_args()
+
+tag_templates = {}
+
+c = csv.DictReader(args.csvfile, delimiter=',', quotechar='"', fieldnames=['tag', 'custom_field_name', 'custom_field_value'])
```

Comment:
probably nicest to drop the arguments where you're just using the defaults, like delimiter and quotechar, right?